packages/example/package.json

```diff
@@ -15,6 +16,7 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "polkadot-snap": "0.0.1",
```

Comment:
I dont' think we need that as dependency, probably just @nodefactory/metamask-polkadot-adapter and @nodefactory/metamask-polkadot-types